### DISCUSSION TITLE
can't open twitch streams

### DISCUSSION THREAD

**Discussion by olaakuna** [streamlink/streamlink#5013] [Q&A] [answered]

**Comment by bastimeyer** [comment, marked as answer]
The OAuth token you've set is wrong. Simple as that. And no, don't ever share it, because it grants access to your Twitch account via Twitch's GraphQL API. Remove `--twitch-api-header`, or follow the instructions from the docs and fix the invalid value.
https://streamlink.github.io/cli/plugins/twitch.html

Btw, there's a pull request from just two hours ago which adds better error messages for exactly this case:
https://github.com/streamlink/streamlink/pull/5011
https://github.com/streamlink/streamlink/issues/5009
